<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vRTU Protocol Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>data/racoon.png</normaloff>data/racoon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
      <item row="3" column="0" colspan="3">
       <widget class="QTableWidget" name="tableConnections"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QComboBox" name="cbIpSelect"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="editPort">
        <property name="text">
         <string>2404</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTableView" name="tableConnectionDetails"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableMainLog"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1322</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStartClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartClicked()</slot>
  <slot>onStopClicked()</slot>
 </slots>
</ui>
